admin/ajax.php

```diff
@@ -80,7 +63,6 @@ function wpseo_set_ignore() {
 
 	if ( $ignore_key === 'indexation_warning' ) {
 		WPSEO_Options::set( 'indexing_reason', '' );
-		WPSEO_Options::set( 'indexation_warning_hide_until', false );
 	}
```

Comment:
You can remove this entire if-statement. It isn't used anymore for the indexing part.